Services/Implementations/TaskService.cs

```diff
@@ -0,0 +1,53 @@
+namespace Services.Implementations;
+using Repositories;
+using DataTransfer.Objects;
+
+public class TaskService : ITaskService
+{
+	private readonly ITaskRepository taskRepo;
+
+	public TaskService(ITaskRepository taskRepo)
+	{
+		this.taskRepo = taskRepo;
+	}
+
+	public IEnumerable<TaskDto> GetTasks()
+	{
+		return taskRepo.GetTasks();
+	}
+
+    public IEnumerable<TaskDto> GetTasksByDeadline()
+	{
+		throw new NotImplementedException();
```

Comment:
Implement the method. It should return the TaskDtos ordered by deadline from the closest to the furthest.